src/puppetlabs/puppetdb/config.clj

```diff
@@ -156,7 +156,7 @@
    :subname     (format "file:%s;hsqldb.tx=mvcc;sql.syntax_pgs=true" (io/file vardir "db"))})
 
 (defn validate-db-settings
-  "Throws a {:type ::configuration-error :message m} exception
+  "Throws a {:type ::cli-error :message m} exception
```

Comment:
That would be downright useful. The right way into this would probably be to have the config parser (in trapperkeeper) annotate the parsed config with file/line metadata, like the clojure reader.